pepper-apis/config/squid.conf

```diff
@@ -0,0 +1,22 @@
+# turn off caching
+cache deny all
+cache_dir null /tmp
+
+acl SSL_ports port 443
+# elasticsearch port
+acl Safe_ports port 9243
+acl CONNECT method CONNECT
+http_access deny !Safe_ports
```

Comment:
Do we want to also deny `CONNECT` usage for non-ssl ports?

Reply:
Yup! Just added it.